exporters/variables-scss/src/formatters/stylesFormatter.ts

```diff
@@ -8,20 +8,23 @@ export const formatLinesAtEndOfTheFile = (css: string): string => {
   return css.replace(/\n{2,}$/, '\n');
 };
 
-export const formatCSS = (css: string): string => {
+export const formatStyles = (css: string, hasJsOutput: boolean): string => {
```

Comment:
I think this function does not format only styles anymore. Please, consider a better name.